src/client/core/utils/dom.js

```diff
@@ -443,3 +443,9 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
 
     return null;
 }
+
+export function canAcceptValue (element) {
```

Comment:
@VasilyStrelyaev, in common these elements are input elements (they are placed for taking input data from user).
Can we pick an another common word or words combination for grouping these elements?